packages/patternfly-4/react-core/src/components/Card/Card.tsx

```diff
@@ -11,18 +11,21 @@ export interface CardProps extends React.HTMLProps<HTMLDivElement> {
   component?: React.ReactNode;
   /** Modifies the card to include hover styles on :hover */
   isHoverable?: boolean;
+  isCompact?: boolean; 
```

Comment:
oh woops, looks like we also need a descriptive comment here

Reply:
Nice catch! :)